dor/providers/process_basic_image.py

```diff
@@ -398,6 +402,144 @@ def run(self) -> None:
         )
 
 
+@dataclass
+class ExtractImageTextCoordinates(Operation):
+    language: str = "eng"
+
+    def run(self) -> None:
+        file_info = FileInfo(
+            identifier=self.accumulator.file_set_identifier.identifier,
+            basename=self.accumulator.file_set_identifier.basename,
+            uses=[UseFunction.service, UseFormat.text_coordinates],
+            mimetype=Mimetype.XML.value,
+        )
+
+        service_result_file = self.accumulator.get_file(
+            function=[UseFunction.service], format=UseFormat.image
+        )
+
+        alto_xml = ImageTextExtractor(image_path=service_result_file.file_path, language=self.language).alto
+        alto_file_path = self.accumulator.file_set_directory / file_info.path
+        alto_file_path.write_text(alto_xml)
+
+        try:
+            tech_metadata = TechnicalMetadata.create(alto_file_path)
+        except JHOVEDocError:
+            return None
```

Comment:
We were doing this in the other operations, so I've repeated it here, but wanted to flag as something we may want to revisit. Re-raising as an `OperationError` may give us more control for error recovery, i.e. we could catch it in `process_basic_image` and then delete the partial file set.